packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx

```diff
@@ -136,7 +141,7 @@ function MapsIndoorsMap({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId
         setCurrentCategories(uniqueCategories);
     }
 
-    /*
+    /**
```

Comment:
Not a function definition. No need for JSDoc.

```suggestion
    /*
```